insalan/tournament/payment.py

```diff
@@ -23,7 +23,7 @@ def fetch_registration(product, user):
         """
         Fetch a registration for a user and product.
 
-        Returns a tuple (reg, is_manager), which is pretty explicit. Raises
+        Returns a tuple (reg, is_manager, is_substitute), which is pretty explicit. Raises
```

Comment:
Wouldn't it make more sense to use an "enum" than two booleans?

Reply:
There's a lot of duplicated codes in this file. I think we keep it like that for now and we'll refactor everything later.